### PR TITLE
chore: Added back docker push to environment workflow

### PR DESCRIPTION
**Pull request by jamtur01**
Signed-off-by: James Turnbull <james@lovedthanlost.net>

### CODE REVIEW

File: .github/workflows/environment.yml [hunk 1]
```diff
@@ -25,4 +25,8 @@ jobs:
           sudo apt clean
           docker rmi $(docker image ls -aq)
           df -h
-      - run: make environment-prepare
+      - name: prepare environment
+        run: make environment-prepare
+      - name: push image
+        if: github.event_name != 'workflow_dispatch'
```
Comment (Hoverbear) [inline .github/workflows/environment.yml:31]: So we're adding a dispatch later? https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event

Comment (jamtur01) [reply .github/workflows/environment.yml:31]: there's also a workflow_dispatch in there.

Comment (Hoverbear) [reply .github/workflows/environment.yml:31]: Amazing.

**Review by Hoverbear — APPROVED** [review]

File: .github/workflows/environment.yml (hunk 1 above)
Comment (jamtur01) [reply .github/workflows/environment.yml:31]: :)

**Pull request merged**